domains/pallets/executive/src/lib.rs

```diff
@@ -421,8 +421,8 @@ where
             .zip(new_header.digest().logs().iter());
         for (header_item, computed_item) in items_zip {
             header_item.check_equal(computed_item);
-            assert!(
```

Comment:
How is this relevant in this PR?

Reply:
not relevant might have slipped when I was seeing around. neverthless a better change IMO